mqtt/mqtt-bridge/src/upstream/events/local.rs

```diff
@@ -92,7 +95,7 @@ impl PumpMessageHandler for LocalUpstreamPumpEventHandler {
                 let doc = doc! { "reason": reason };
                 match doc.to_writer(&mut payload) {
                     Ok(_) => Some(Publication {
-                        topic_name: format!("$edgehub/rpc/nack/{}", command_id),
+                        topic_name: format!("$upstream/rpc/nack/{}", command_id),
```

Comment:
What was the reason this changed to downstream?

Reply:
we discussed with @vipeller that it will cleaner if all requests that go to upstream broker have `$upstream` prefix, all responses `$downstream`. this way it is easier to reason about who is the recepient of the message.

Reply:
oh, actually you spotted an error here!

Reply:
On the line for my comment we are sending a publish to $upstream only in the case of nack. This is a response so shouldn't it be $downstream?

Reply:
You wrote the test to match this upstream / downstream pattern so I think I am missing something

Reply:
Your comment didn't load for me. I see it now and understand.